public/components/custom_panels/home.tsx

```diff
@@ -9,8 +9,8 @@ import { Toast } from '@elastic/eui/src/components/toast/global_toast_list';
 import _ from 'lodash';
 import React, { ReactChild, useState } from 'react';
 // eslint-disable-next-line @osd/eslint/module_migration
```

Comment:
This may not be related to this PR, but I wondering if we could remove these rules as long as you don't have issue with these rules

Reply:
I haven’t checked if its currently in use, will remove it if its not used